exercises/E-strings-concatenation/exercise.js

```diff
@@ -1,3 +1,5 @@
 // Start by creating a variable `message`
-
-console.log(message);
+let greetingStart = 'Hello, my name is '
+let name = 'Sharm'
```

Comment:
It's a small thing here, but try to get into the habit of assigning variables that you're not planning to change as const instead of let, you'll understand later when you get onto scope but feel free to message me if you don't understand